repo2rocrate could not be found in the source repo, GitHub project nor PyPi
author: John Dey Fred Hutch
diff -ruN nf_core-3.3.1.orig/requirements.txt nf_core-3.3.1/requirements.txt
--- nf_core-3.3.1.orig/requirements.txt	2025-06-02 07:26:17.000000000 -0700
+++ nf_core-3.3.1/requirements.txt	2025-07-02 14:39:43.764374000 -0700
@@ -18,7 +18,6 @@
 rich-click==1.8.*
 rich>=13.3.1
 rocrate
-repo2rocrate
 tabulate
 textual==2.1.2
 trogon
